app/views/citizens/check_answers/_bank_accounts.html.erb

```diff
@@ -1,28 +1,28 @@
-<h2 class="govuk-heading-l"><%= t("citizens.check_answers.index.bank_accounts.heading") %></h2>
+<h2 class="govuk-heading-l"><%= t(".heading") %></h2>
 
-<%- @legal_aid_application.applicant.bank_accounts.each_with_index do |account, index| %>
-  <h2 class="govuk-heading-m govuk-!-padding-top-4"><%= t("citizens.check_answers.index.bank_accounts.account") %> <%= index + 1 %></h2>
-
-  <%= govuk_summary_list(actions: false, classes: "govuk-!-padding-bottom-4") do |summary_list| %>
-    <%= summary_list.with_row do |row| %>
-      <%= row.with_key { t("citizens.check_answers.index.bank_accounts.bank") } %>
-      <%= row.with_value { account.bank_provider.name } %>
-    <% end %>
-    <%= summary_list.with_row do |row| %>
-      <%= row.with_key { t("citizens.check_answers.index.bank_accounts.account_number") } %>
-      <%= row.with_value { account.account_number } %>
-    <% end %>
-    <%= summary_list.with_row do |row| %>
-      <%= row.with_key { t("citizens.check_answers.index.bank_accounts.sort_code") } %>
-      <%= row.with_value { account.sort_code } %>
-    <% end %>
-    <%= summary_list.with_row do |row| %>
-      <%= row.with_key { t("citizens.check_answers.index.bank_accounts.balance") } %>
-      <%= row.with_value { value_with_currency_unit(account.balance, account.currency) } %>
+<% @legal_aid_application.applicant.bank_accounts.each_with_index do |account, index| %>
+  <%= govuk_summary_card(title: t(".account", index: index + 1)) do |card| %>
```

Comment:
I believe we agreed to use the single erb block approach for summary cards where possible